sdk/schemaregistry/schema-registry/recordings/browsers/schemaregistryclient/recording_fails_to_get_schema_id_when_given_invalid_args.json

```diff
@@ -1,67 +1,99 @@
-    "strict-transport-security": "max-age=31536000; includeSubDomains",
-    "x-content-type-options": "nosniff",
-    "x-ms-ests-server": "2.1.12231.7 - WUS2 ProdSlices",
-    "x-ms-request-id": "19fae638-b9c4-4c2b-807b-77e139ad0d00"
-   }
-  },
-  {
-   "method": "POST",
-   "url": "https://endpoint/$schemaGroups/group-1/schemas/azsdk_js_test:get-id",
-   "query": {
-    "api-version": "2021-10"
-   },
-   "requestBody": "{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"com.azure.schemaregistry.samples\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favoriteNumber\",\"type\":\"int\"}]}",
-   "status": 415,
-   "response": "{\"Code\":415,\"Detail\":\"Invalid schema type for POST request. 'null' is not supported. TrackingId:7fe9dcfb-0fca-4ab2-9beb-473aa5fe6406_G23, SystemTracker:endpoint:$schemaGroups\\/group-1\\/schemas\\/azsdk_js_test:get-id, Timestamp:2021-11-10T07:53:32\"}",
-   "responseHeaders": {
-    "content-type": "application/json",
-    "date": "Wed, 10 Nov 2021 07:53:31 GMT",
-    "server": "Microsoft-HTTPAPI/2.0",
-    "strict-transport-security": "max-age=31536000",
-    "transfer-encoding": "chunked"
-   }
-  },
-  {
-   "method": "POST",
-   "url": "https://endpoint/$schemaGroups/group-1/schemas/azsdk_js_test:get-id",
-   "query": {
-    "api-version": "2021-10"
-   },
-   "requestBody": "{\"type\":\"record\",\"name\":\"User\",\"namespace\":\"com.azure.schemaregistry.samples\",\"fields\":[{\"name\":\"name\",\"type\":\"string\"},{\"name\":\"favoriteNumber\",\"type\":\"int\"}]}",
-   "status": 415,
-   "response": "{\"Code\":415,\"Detail\":\"Invalid schema type for POST request. 'not-valid' is not supported. TrackingId:1111c7fa-5159-4fcd-9530-a48bc1602146_G23, SystemTracker:endpoint:$schemaGroups\\/group-1\\/schemas\\/azsdk_js_test:get-id, Timestamp:2021-11-10T07:53:32\"}",
-   "responseHeaders": {
-    "content-type": "application/json",
-    "date": "Wed, 10 Nov 2021 07:53:32 GMT",
-    "server": "Microsoft-HTTPAPI/2.0",
-    "strict-transport-security": "max-age=31536000",
-    "transfer-encoding": "chunked"
-   }
-  }
- ],
- "uniqueTestInfo": {
-  "uniqueName": {},
-  "newDate": {}
- },
- "hash": "36009d8faaaf7d79236f8a74fecd05d9"
-}
\ No newline at end of file
+  "Entries": [
+    {
+      "RequestUri": "https://endpoint/$schemaGroups/group-1/schemas/azsdk_js_test:get-id?api-version=2021-10",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip, deflate, br",
+        "Accept-Language": "en-US",
+        "Authorization": "Sanitized",
+        "Connection": "keep-alive",
+        "Content-Length": "160",
+        "Content-Type": "application/json; serialization=null",
+        "Referer": "http://localhost:9876/",
+        "sec-ch-ua": "",
+        "sec-ch-ua-mobile": "?0",
+        "sec-ch-ua-platform": "",
+        "Sec-Fetch-Dest": "empty",
+        "Sec-Fetch-Mode": "cors",
+        "Sec-Fetch-Site": "same-site",
```

Comment:
I am not familiar with those headers, who add them to the request? I did a quick grep in our core packages and did not find any reference to them, so I suspect it is the proxy tool itself that is adding them. I think some of those values are not valid. For instance, for `sec-ch-ua`, it does not follow the syntax in https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-CH-UA#examples. I do not think it is that important, but I thought it may be worth looking into.

Reply:
Coming from karma. 
Request from karma goes to the proxy tool, and the proxy-tool makes the request to the service.

Reply:
Oh I see, thanks!